base/cvd/cuttlefish/host/commands/cvd/cli/selector/arguments_lexer.cpp

```diff
@@ -206,5 +291,16 @@ Result<std::vector<std::string>> ArgumentsLexer::Preprocess(
   return new_args;
 }
 
+Result<std::deque<ArgToken>> TokenizeArguments(
+    const std::vector<std::string>& args) {
+  ArgumentsLexerBuilder builder;
+
+  std::unique_ptr<ArgumentsLexer> lexer = CF_EXPECT(builder.Build());
+  CF_EXPECT(lexer.get());
+
+  std::vector<ArgToken> as_vector = CF_EXPECT(lexer->Tokenize(args));
+  return std::deque<ArgToken>(as_vector.begin(), as_vector.end());
```

Comment:
nit: I actually prefer this function to return a `vector` instead of a `deque`. There is no difference performance-wise, it's just that `deque` is used for a very specific purpose (efficient insertion and deletion at the front) which is not needed here. I prefer to introduce the `deque` at the caller site where the need for it is evident.

Reply:
Done.